packages/database/src/migrations/20231123022242-AddEmailAdminPermissionGroupsToDashboardMailingList-modifies-schema.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    ALTER TABLE dashboard_mailing_list ADD COLUMN email_admin_permission_groups TEXT[] NOT NULL DEFAULT '{}';
```

Comment:
"mailing_list_admin_permission_groups"? To me email is a bit vauge.

Reply:
I like the rename suggestion. Think `admin_permission_groups` would be too vague? Given they're a column of the `dashboard_mailing_list` I feel the context could be implied.

Reply:
Perfect!